store/sqlstore/channel_store.go

```diff
@@ -1205,6 +1205,25 @@ func (s SqlChannelStore) GetMembers(channelId string, offset, limit int) store.S
 	})
 }
 
+func (s SqlChannelStore) GetChannelMembersTimezones(channelId string) store.StoreChannel {
+	return store.Do(func(result *store.StoreResult) {
+		var dbMembersTimezone []map[string]string
+		_, err := s.GetReplica().Select(&dbMembersTimezone,
+			`SELECT
+					Users.Timezone
+				FROM
+					ChannelMembers
+				LEFT JOIN
+					Users  ON ChannelMembers.UserId = Id
+				WHERE ChannelId = :ChannelId`, map[string]interface{}{"ChannelId": channelId})
+		if err != nil {
+			result.Err = model.NewAppError("SqlChannelStore.GetChannelMembersTimezones", "store.sql_channel.get_members.app_error", nil, "channel_id="+channelId+err.Error(), http.StatusInternalServerError)
```

Comment:
`err.Error()` is being concatenated to the `channel_id=<channelId>` string here without a delimiter. It seems like we use commas elsewhere in this code?

Reply:
done and fix in the other function as well